setup.cfg

```diff
@@ -21,9 +21,11 @@ defines = HAVE_SASL HAVE_TLS HAVE_LIBLDAP_R
 extra_compile_args =
 extra_objects =
 
+# Uncomment this if your libldap is not thread-safe and you need libldap_r
+# instead
```

Comment:
setup.cfg changes seem unrelated to this PR

Reply:
If we keep the line there, people's builds with 2.5+ will keep failing, right?